<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="history_label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;历史&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="output_label">
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;结果&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="2">
        <widget class="QPushButton" name="input_20">
         <property name="text">
          <string>PI</string>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QPushButton" name="input_6">
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="6" column="5">
        <widget class="QPushButton" name="input_3">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="8" column="6">
        <widget class="QPushButton" name="input_equal">
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="input_4">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="8" column="5">
        <widget class="QPushButton" name="input_point">
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QPushButton" name="input_multiply">
         <property name="text">
          <string>×</string>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QPushButton" name="input_2">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="input_16">
         <property name="text">
          <string>√x</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="input_22">
         <property name="text">
          <string>x²</string>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QPushButton" name="input_minus">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="input_15">
         <property name="text">
          <string>)</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QPushButton" name="input_negate">
         <property name="text">
          <string>+/-</string>
         </property>
        </widget>
       </item>
       <item row="6" column="6">
        <widget class="QPushButton" name="input_add">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="input_12">
         <property name="text">
          <string>10^x</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="input_8">
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QPushButton" name="input_1">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="input_21">
         <property name="text">
          <string>1/x</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="input_7">
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QPushButton" name="input_5">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QPushButton" name="input_17">
         <property name="text">
          <string>ln</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="input_divide">
         <property name="text">
          <string>÷</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="input_18">
         <property name="text">
          <string>c</string>
         </property>
        </widget>
       </item>
       <item row="8" column="4">
        <widget class="QPushButton" name="input_0">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="input_9">
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="input_13">
         <property name="text">
          <string>log</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="input_10">
         <property name="text">
          <string>n!</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="input_14">
         <property name="text">
          <string>(</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="input_19">
         <property name="text">
          <string>e</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="DEG">
         <property name="text">
          <string>DEG</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QRadioButton" name="RAD">
         <property name="text">
          <string>RAD</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="input_power">
         <property name="text">
          <string>x^y</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="CalculateMenu">
    <property name="title">
     <string>计算</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>三角函数</string>
     </property>
     <widget class="QMenu" name="menunormal">
      <property name="title">
       <string>普通</string>
      </property>
      <addaction name="actionsin"/>
      <addaction name="actioncos"/>
      <addaction name="actiontan"/>
      <addaction name="actionsec"/>
      <addaction name="actioncsc"/>
      <addaction name="actioncot"/>
     </widget>
     <widget class="QMenu" name="menu_2">
      <property name="title">
       <string>双曲</string>
      </property>
      <addaction name="actionsinh"/>
      <addaction name="actioncosh"/>
      <addaction name="actiontanh"/>
      <addaction name="actionsech"/>
      <addaction name="actioncsch"/>
      <addaction name="actioncoth"/>
     </widget>
     <addaction name="menunormal"/>
     <addaction name="separator"/>
     <addaction name="menu_2"/>
    </widget>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>记忆</string>
     </property>
     <addaction name="actionm"/>
     <addaction name="actionm_2"/>
     <addaction name="actionms"/>
    </widget>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>其他函数</string>
     </property>
     <addaction name="action_up"/>
     <addaction name="action_low"/>
     <addaction name="action_abs"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="menu_4"/>
    <addaction name="menu_3"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="HelpMenu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="CalculateMenu"/>
   <addaction name="HelpMenu"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionsin">
   <property name="text">
    <string>sin</string>
   </property>
  </action>
  <action name="actioncos">
   <property name="text">
    <string>cos</string>
   </property>
  </action>
  <action name="actiontan">
   <property name="text">
    <string>tan</string>
   </property>
  </action>
  <action name="actionsec">
   <property name="text">
    <string>sec</string>
   </property>
  </action>
  <action name="actioncsc">
   <property name="text">
    <string>csc</string>
   </property>
  </action>
  <action name="actioncot">
   <property name="text">
    <string>cot</string>
   </property>
  </action>
  <action name="actionsinh">
   <property name="text">
    <string>sinh</string>
   </property>
  </action>
  <action name="actioncosh">
   <property name="text">
    <string>cosh</string>
   </property>
  </action>
  <action name="actiontanh">
   <property name="text">
    <string>tanh</string>
   </property>
  </action>
  <action name="actionsech">
   <property name="text">
    <string>sech</string>
   </property>
  </action>
  <action name="actioncsch">
   <property name="text">
    <string>csch</string>
   </property>
  </action>
  <action name="actioncoth">
   <property name="text">
    <string>coth</string>
   </property>
  </action>
  <action name="actionm">
   <property name="text">
    <string>M+</string>
   </property>
  </action>
  <action name="actionm_2">
   <property name="text">
    <string>M-</string>
   </property>
  </action>
  <action name="actionms">
   <property name="text">
    <string>MS</string>
   </property>
  </action>
  <action name="action_up">
   <property name="text">
    <string>向上取整</string>
   </property>
  </action>
  <action name="action_low">
   <property name="text">
    <string>向下取整</string>
   </property>
   <property name="iconText">
    <string>向下取整</string>
   </property>
  </action>
  <action name="action_abs">
   <property name="text">
    <string>绝对值</string>
   </property>
   <property name="iconText">
    <string>绝对值</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
